src/customFileParser/unpackGeometry/main.ts

```diff
@@ -78,15 +81,17 @@ function unpackFilePrimitive(
   primitiveCompressedData: CompressedGeometryData,
   uncompressedValues: any,
   treeIndexNodeIdMap: any,
-  colorMap: any) {
+  colorMap: any,
```

Comment:
Why is this any? We have a `ColorMap` type.

Reply:
Fixed